classes/patterns/class-sideimagewithtextandcta.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Side image with text and CTA pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Side image with text and CTA.
+ *
+ * @package P4GBKS\Patterns
+ */
+class SideImageWithTextAndCta {
+
+	/**
+	 * Pattern name.
+	 *
+	 * @const string PATTERN_NAME.
+	 */
+	const PATTERN_NAME = 'side-image-with-text-and-cta';
+
+	/**
+	 * Side image with text and CTA pattern constructor.
+	 */
+	public function __construct() {
+		if ( ! function_exists( 'register_block_pattern' ) ) {
+			return;
+		}
+
+		register_block_pattern(
+			self::PATTERN_NAME,
+			[
+				'title'      => __( 'Side image with text and CTA', 'planet4-blocks-backend' ),
+				'keywords'   => [ 'cta', 'side', 'image' ],
+				'categories' => [ 'planet4' ],
+				'content'    => '
+					<!-- wp:media-text -->
```

Comment:
WP seems to add the missing `className` attribute here after inserting. But probably best to add it here anyway.